devops/test_configs.json

```diff
@@ -2,22 +2,22 @@
   "lts": [
```

Comment:
lts? Still?

Reply:
Unrelated :) I'm planning on dropping this json file altogether though.